server/resources/email/activation_email.html

```diff
@@ -1,15 +1,147 @@
+        padding-inline: 50px;
+        margin-block: 50px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        overflow-wrap: break-word;
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+      }
+
+      .button a {
+        margin-block: 15px;
+        padding: 14px;
+        background-color: hsl(121, 64%, 17%);
+        color: white;
+        border: 0px;
+        border-radius: 15px;
+        text-decoration: none;
+      }
+
+      .button a:hover {
+        cursor: pointer;
+        background-color: hsl(121, 64%, 28%);
+      }
+
+      .button {
+        text-align: center;
+        max-width: 40vw;
+        padding-block: 15px;
+      }
+
+      .main {
+        max-width: 40vw;
+        margin-left: 30vw;
+        border: 1px solid hsl(96, 50%, 37%);
+        border-radius: 10px;
+      }
+
+      .end p {
+        margin: 0 0 0 20px;
+      }
+
+      .end {
+        padding-bottom: 10px;
+      }
+
+      .link,
+      .text {
+        text-align: center;
+        padding-inline: 15px;
+      }
+
+      .link {
+        margin-bottom: 30px;
+        padding-inline: 50px;
+        max-width: 40vw;
+      }
+
+      .linkText {
+        overflow: hidden;
+        overflow-wrap: break-word;
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+      }
+
+      /*Mobile*/
+      @media only screen and (max-width: 600px) {
+        .main {
+          max-width: 90vw;
+          margin-left: 2.5vw;
+        }
+
+        .button,
+        .link {
+          max-width: 90vw;
+        }
+
+        .button a:active {
+          background-color: hsl(121, 64%, 28%);
+        }
+
+        .button a {
+          padding: 10px;
+        }
+
+        h1 {
+          padding-inline: 25px;
+        }
+      }
+
+      /*Laptop and Tablet*/
+      @media only screen and (min-width: 600px) and (max-width: 1200px) {
+        .main {
+          max-width: 60vw;
+          margin-left: 20vw;
+        }
+
+        .button,
+        .link {
+          max-width: 60vw;
+        }
+
+        .button a:active {
+          background-color: hsl(121, 64%, 28%);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="main">
+      <h1>Üdvözlünk {{ username }}!</h1>
+      <p class="text">
+        Kérem erősítse meg a Veryrezsi felhasználói fiókjának regisztrációját! A
+        regisztráció megerősítéséhez kérem kattintson az alábi gombra.
+      </p>
+      <div class="button">
+        <a href="{{ activation_link }}">Regisztráció megerősítése</a>
+      </div>
+      <p class="text">
+        Vagy használja az alábbi linket ha a gomb nem jelenne meg vagy nem
+        müködne:
+      </p>
+      <p class="link">
+        <a class="linkText" href="{{ activation_link }}"
+          >{{ activation_link }}</a
+        >
+      </p>
+      <div class="end">
+        <p>Üdvözlettel,</p>
+        <p>A Veryrezsi csapata!</p>
```

Comment:
I don't really care about the language it is in, right now.
Just make sure you settled on Hungarian. Do not change it everytime.

Reply:
Whoops that is a bummer, I didn't not write it btw, but still I did not care to consider it 😄 Will fix.

Reply:
ChatGPT helped us out in translating 😄

Reply:
Perfect.